Documentation/ceph-object.md

```diff
@@ -242,8 +242,42 @@ cat /tmp/rookObj-download
 
 ## Access External to the Cluster
 
-Rook sets up the object storage so pods will have access internal to the cluster. If your applications are running outside the cluster,
-you will need to setup an external service through a `NodePort`.
+There are two possible approaches:
+
+### Port-Forward
```

Comment:
This section assumes the aws environment. Since there are so many different environments, we have tried to avoid documentation for specific environments. Can we make this more generic? It's ok to link to aws docs as an example, but I don't think we want a whole section of the doc that only works for aws.